Dockerfile.withset

```diff
@@ -1,9 +1,9 @@
 # This "FROM" image is previously emitted by nix
 FROM ghcr.io/engineer-man/piston:base-latest
 
-ENV PISTON_FLAKE_PATH=/piston/packages
-COPY runtimes/ /piston/packages/runtimes
-COPY flake.nix flake.lock /piston/packages/
+ENV PISTON_FLAKE_PATH=/piston/src
+COPY runtimes/ /piston/src/runtimes
+COPY flake.nix flake.lock /piston/src/
```

Comment:
What was wrong with `/piston/packages`?

Reply:
I found it to be a bit misleading since this directory also contains the source code.